bundles/EcommerceFrameworkBundle/Model/AbstractOrder.php

```diff
@@ -71,23 +72,21 @@ public function setSubTotalPrice($subTotalPrice)
     }
 
     /**
-     * Should return a float
+     * @throws UnsupportedException
      *
-     * @return void
+     * @return float
      */
     public function getSubTotalNetPrice()
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('getSubTotalNetPrice not implemented for ' . get_class($this));
+        throw new UnsupportedException('getSubTotalNetPrice not implemented for ' . get_class($this));
```

Comment:
As this breaks as well, we shouldn't do this in this PR.
I think we should also think about making the method abstract or using interfaces instead.

Reply:
https://github.com/pimcore/pimcore/pull/7553